src/main/scala/firrtl/passes/Checks.scala

```diff
@@ -51,6 +51,8 @@ object CheckHighForm extends Pass {
     s"$info: A single module must be named $name.")
   class NegArgException(info: Info, mname: String, op: String, value: Int) extends PassException(
     s"$info: [module $mname] Primop $op argument $value < 0.")
+  class LsbLargerThanMsb(info: Info, mname: String, op: String, lsb: Int, msb: Int) extends PassException(
```

Comment:
Should this be LsbLargerThanMsbException? It seems like the convention to and the "Exception" suffix to classes that extend Exception.

Reply:
Good catch, thanks. New PR inbound...